python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -175,158 +175,288 @@ class InsertFakesConnections(PipelineTaskConnections,
 class InsertFakesConfig(PipelineTaskConfig,
                         pipelineConnections=InsertFakesConnections):
     """Config for inserting fake sources
-
-    Notes
-    -----
-    The default column names are those from the University of Washington sims database.
     """
 
-    raColName = pexConfig.Field(
-        doc="RA column name used in the fake source catalog.",
+    # Unchanged
+
+    doCleanCat = pexConfig.Field(
+        doc="If true removes bad sources from the catalog.",
+        dtype=bool,
+        default=True,
+    )
+
+    fakeType = pexConfig.Field(
+        doc="What type of fake catalog to use, snapshot (includes variability in the magnitudes calculated "
+            "from the MJD of the image), static (no variability) or filename for a user defined fits"
+            "catalog.",
         dtype=str,
-        default="raJ2000",
+        default="static",
     )
 
-    decColName = pexConfig.Field(
-        doc="Dec. column name used in the fake source catalog.",
+    calibFluxRadius = pexConfig.Field(
+        doc="Aperture radius (in pixels) that was used to define the calibration for this image+catalog. "
+        "This will be used to produce the correct instrumental fluxes within the radius. "
+        "This value should match that of the field defined in slot_CalibFlux_instFlux.",
+        dtype=float,
+        default=12.0,
+    )
+
+    coaddName = pexConfig.Field(
+        doc="The name of the type of coadd used",
         dtype=str,
-        default="decJ2000",
+        default="deep",
     )
 
-    doCleanCat = pexConfig.Field(
-        doc="If true removes bad sources from the catalog.",
+    doSubSelectSources = pexConfig.Field(
+        doc="Set to True if you wish to sub select sources to be input based on the value in the column"
+            "set in the sourceSelectionColName config option.",
         dtype=bool,
-        default=True,
+        default=False
```

Comment:
Are you changing the default value here?

Reply:
No, I think github (or diff) just had a hard time figuring out which two lines to compare.  The "before" side is for `doCleanCat`, which is still True, and the "after" side is for `doSubSelectSources`, which is still False.